dashboard/src/components/Header/Menu.tsx

```diff
@@ -31,28 +31,29 @@ function Menu({ clusters, appVersion, logout }: IContextSelectorProps) {
   const ref = useRef(null);
   useOutsideClick(setOpen, [ref], open);
 
-  const getTheme = (isDark: boolean): SupportedThemes =>
-    isDark ? SupportedThemes.dark : SupportedThemes.light;
-
-  const [isDarkModeEnabled, setIsDarkModeEnabled] = useState(
-    localStorage.getItem("theme") === SupportedThemes.dark ? true : false,
+  const [theme, setTheme] = useState<string>(
+    localStorage.getItem("theme") !== undefined
+      ? localStorage.getItem("theme") || SupportedThemes.light
+      : SupportedThemes.light,
   );
```

Comment:
```
const [theme, setTheme] = useState(localStorage.getItem("theme") || SupportedThemes.light)
```
(if getItem returns "" when not defined it will use the light theme)